main/src/ca/uwaterloo/flix/language/ast/NamedAst.scala

```diff
@@ -18,15 +18,18 @@ package ca.uwaterloo.flix.language.ast
 
 import ca.uwaterloo.flix.language.ast
 import ca.uwaterloo.flix.language.ast.Ast.{Denotation, Source}
-import ca.uwaterloo.flix.util.collection.MultiMap
 
 import scala.collection.immutable.List
 
 object NamedAst {
 
+  sealed trait DefOrSig {
+    val loc: SourceLocation
+  }
+
   case class Root(classes: Map[Name.NName, Map[String, NamedAst.Class]],
                   instances: Map[Name.NName, Map[String, List[NamedAst.Instance]]],
-                  defs: Map[Name.NName, Map[String, NamedAst.Def]],
+                  defsAndSigs: Map[Name.NName, Map[String, NamedAst.DefOrSig]],
```

Comment:
Maybe we need a common name for a Def or Sig?

Reply:
Yes, that would be nice. Any ideas?

Reply:
Not really. `Member?` but I dont like it.